application/src/test/java/uk/nhs/hee/tis/revalidation/service/RecommendationServiceTest.java

```diff
@@ -825,10 +848,13 @@ void shouldGetRecommendationStatusCheckDtos() {
     final var recommendation = buildRecommendation(gmcId, recommendationId, SUBMITTED_TO_GMC,
         UNDER_REVIEW);
 
-    when(recommendationRepository.findAllByRecommendationStatus(RecommendationStatus.SUBMITTED_TO_GMC))
+    when(recommendationRepository
```

Comment:
The changes which are seen next are produced due to the "reformat code" button. They can be ignored.